auth/auth.go

```diff
@@ -1,4 +1,4 @@
-package server
+package auth
```

Comment:
It's unfortunate this needs to be moved at all. Ideally, authentication to the website should live exclusively in package server. One way of doing this could be to create or update a server API which will proxy or redirect the request to upstream. This would be useful not just for the windows and mac app but for version checking in instances such as linux or the mac os non-app binary